infra/README.md

```diff
@@ -98,6 +98,45 @@ You can do the same with another VM or a different username:
 source ./ansible/scripts/get-vm-secrets.sh "tournesol-vm" "jst"
 ```
 
+## Accessing the back end admin interface
+
+To protect the back end's admin interface, its access is restricted to
+HTTP requests coming from its own host.
+
+Users having an account on the server can access this interface by using an
+SSH tunnel.
+
+### Example: accessing the staging admin
+
+First, edit your own `/etc/hosts` or equivalent. This is required to make our
+requests reach the correct NGINX virtual host, and not the default one.
+
+```
+127.0.0.1    api.staging.tournesol.app
```

Comment:
This doesn't look sustainable on the long term, as resolving the back end to localhost will prevent us to use the front end...

Reply:
various solutions to these issues posted to #infrastructure Discord channel

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1094/commits/2e247199a4f96f22c359039c02ae98a8dc3a410f